.storybook/webpack.config.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
+
+module.exports = (storybookBaseConfig, configType) => {
+  storybookBaseConfig.module.rules.push(
+    {
+      test: /\.scss$/,
+      loaders: ['style-loader', 'css-loader?modules', 'sass-loader'],
+      include: path.resolve(__dirname, '..', 'src'),
+    },
+    {
+      test: /\.(png|jpg|gif)$/,
+      use: [
+        {
+          loader: 'file-loader',
+        },
+      ],
+    },
+  );
+
+  return storybookBaseConfig;
```

Comment:
We might be able to resolve the `src` folder here, right? Adding the following code seems to work 👌

```
storybookBaseConfig.resolve.modules.push('src');
```

Reply:
I will give this a try